pkg/tailfile/tailfile.go

```diff
@@ -56,7 +58,10 @@ func TailFile(f io.ReadSeeker, n int) ([][]byte, error) {
 		if cnt > n {
 			break
 		}
-		block--
+		block = 2 * block
+		if block < maxBlock {
```

Comment:
If you feel confused about it, I could change the maxBlock to one positive value, and change the code.